ci/build-project.sh

```diff
@@ -30,6 +30,9 @@ pushd "$(dirname "$0")"
     mkdir -p "${TEST_REPO_PATH}/Game/Plugins"
     cp -R "${GDK_HOME}" "${TEST_REPO_PATH}/Game/Plugins/UnrealGDK"
 
+    # Unreal likes to save some settings outside of the project folders. Let's clean this up to make sure it doesn't cause issues when running the tests.
+    rm -rf ~/Library/Preferences/Unreal\ Engine/
```

Comment:
Interesting, does it do something similar on Windows as well?
Maybe that would explain why we saw so many wrong netmode issues.

Reply:
turns out, yes! they would be in a folder similar to that: `C:\Users\<username>\AppData\Local\<ProjectName>\Saved\Config`

Can add a line to delete that folder as well

Reply:
actually, will do that in a separate pr just to keep both small.